tsconfig.json

```diff
@@ -16,5 +16,6 @@
 		"jsxImportSource": "@emotion/react",
 		"incremental": true
 	},
+	"include": ["**/*", ".storybook/**/*"],
```

Comment:
Rather than `**/*`, reckon we could enumerate the different dirs?

```
"include": ["packages/**/*", "docs/**/*", ".storybook/**/*"],
```

Reply:
We only need to manually specify the storybook folder because it starts with a `.`

All other folders are captured with the first glob 👍